src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SEVEN_DAYS_DISCOUNT = 50;
+  const THREE_DAYS_DISCOUNT = 20;
+  const SEVEN_DAYS = 7;
```

Comment:
```suggestion
  const BIG_DISCOUNT_PERIOD = 7;
```
Не привязывайся к значению переменной. Если завтра скидка будет наступать после 8 дней, а не 7?